addons/hr_org_chart/controllers/hr_org_chart.py

```diff
@@ -22,8 +22,9 @@ def _check_employee(self, employee_id, **kw):
 
         Employee = request.env['hr.employee.public'].with_context(allowed_company_ids=cids)
         employee = Employee.browse(employee_id)
-        # check and raise
-        return employee if employee.has_access('read') else None
+        if check_read_access:
+            return employee if employee.has_access('read') else None
```

Comment:
We could maybe return a empty recordset 🤷🏼‍♂️ 
```suggestion
    return employee if employee.has_access('read') else Employee.browse()
```